src/docs/product/alerts-notifications/issue-alerts.mdx

```diff
@@ -0,0 +1,65 @@
+
+![The alert condition in this example shows the conditions are conflicting with each other.](alert-conditions2.png)
+
+In the example, the alert will not fire because an event **cannot satisfy both** of these conditions simultaneously: Sentry detects/creates a new issue, _and_ the issue has happened ten times.
+
+Types of alert conditions:
+
+| Type | Condition |
+| -- | -- |
+| Issue detection | Sentry detects a new issue |
+| Issue creation | Sentry creates an issue |
+| Issue changes state | `resolved` to `unresolved` |
+| Issue changes state | `ignored` to `unresolved` |
+| Issue changes frequency | An issue has happened more than {**value**} times in {**interval**}. A **value** is a positive integer. An **integer** can be one minute, one hour, one day, one week, or 30 days |
+| Issue visible to users | An issue has been seen by more than {**value**} users in {**interval**}. A **value** is a positive integer. An **interval** is one minute, one hour, one day, one week, or 30 days. |
+| New event detection | Sentry detects a new event |
+| New event detection | Sentry detects a new event with tags matching {**key**} {**match**} {**value**}. A **key**: any tag. A **match**: equals, does not equal, starts with, ends with, contains, does not contain, is set, or is not set. A **value**: any key’s value |
+| New event detection | Sentry detects a new event with {**attribute**} {**match**} {**value**}. An **attribute**: `message`, `platform`, `environment`, `type`, `exception.type`, `exception.value`, `user.id`, `user.email`, `user.username`, `user.ip_address`, `http.method`, `http.url`, `stacktrace.code`, `stacktrace.module`, or `stacktrace.filename`. A **match**: equals, does not equal, starts with, ends with, contains, does not contain, is set, or is not set. A **value**: any attribute’s value. NOTE: When viewing an event, if an attribute is unclear or missing, you can view its JSON payload to determine its value. |
+| New event detection | Sentry detects a new event with level {**match**} {**level**}. A **match**: equal to, less than or equal to, greater than or equal to. A **level**: fatal, error, warning, info, debug, or sample. |
+
+## Actions
+
+Actions are the configurations that determine what will happen when an alert is triggered. The following actions are available:
+
+- Send an email to either [Issue Owners](/product/error-monitoring/issue-owners/), Team, or [Member](/accounts/membership/#member).
+- Send a notification to an [integration](/product/integrations/), which includes [PagerDuty](/product/integrations/integrations/pagerduty/), [Slack](/product/integrations/integrations/slack/), and [Microsoft Teams](/product/integrations/integrations/msteams/). If no legacy integrations or integrations built using the integration platform are enabled, this option is hidden.
```

Comment:
We should call out the options explicitly
<img width="297" alt="Screen Shot 2020-09-29 at 8 59 01 PM" src="https://user-images.githubusercontent.com/693121/94641628-a0241000-0296-11eb-9cee-688e3808f971.png">

For the "send a notification via an integration" option, mention that it includes legacy integrations + integrations built on the integration platform.